### PR TITLE
brew-mas: add page

### PR DESCRIPTION
**Pull request by cfsanderson**
----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/osx/brew-mas.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Install or update an application, not for the first time (apps must already be in the Purchased tab of the App Store):
+
+`mas install {{app_name}} {{app_identifier}}`
+
+- Show all installed applications and their product identifiers:
+
+`mas list`
+
+- List installed apps with pending updates.
```
Comment (pxgamer) [inline pages/osx/brew-mas.md]: Could you update this to use a colon instead of a full stop for the description?

File: pages/osx/brew-mas.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# brew mas
+
+> [mas-cli](https://github.com/mas-cli/mas) is a simple command line interface for the Mac App Store. Designed for scripting and automation.
```
Comment (pxgamer) [inline pages/osx/brew-mas.md]: @agnivade Are markdowns links supported?

File: pages/osx/brew-mas.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Search the Mac App Store by app name and return matching identifiers:
+
+`mas search {{app_name}}`
+
+- Install or update an application, not for the first time (apps must already be in the Purchased tab of the App Store):
```
Comment (pxgamer) [inline pages/osx/brew-mas.md]: Could you maybe simplify this sentence?

Perhaps something such as
>Install or update a previously purchased application:

File: pages/osx/brew-mas.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+`mas upgrade`
+
+- Install updates for a specific app:
+
+`mas upgrade {{app_identifier}}`
+
+- Reset mas:
+
+`mas reset`
+
```
Comment (pxgamer) [inline pages/osx/brew-mas.md]: Looks like there's also some trailing whitespace at the bottom which is being picked up by the linter.

**Comment by cfsanderson** [comment]
Thanks for the corrections @pxgamer - I think it should be in order now unless the link is not supported. Happy to remove it if not.

**Comment by pxgamer** [comment]
Looks fine to me. 👍 Just waiting for Travis which has been slow today.

**Comment by pxgamer** [comment]
Hi @cfsanderson, I've asked and it looks like links aren't supported at the moment. Do you mind removing it for now? Cheers.

**Comment by cfsanderson** [comment]
Sorry for the game of PR tag @pxgamer! I took the opportunity to simplify the description as well as remove the link.

**Comment by pxgamer** [comment]
Looks good to me. 👍

File: pages/osx/brew-mas.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- Install all pending updates:
+
+`mas upgrade`
+
+- Install updates for a specific app:
+
+`mas upgrade {{app_identifier}}`
+
+- Reset mas:
```
[the remark this answers is not in the document]
Comment (cfsanderson) [reply pages/osx/brew-mas.md]: That was actually a mistake. I'm so sorry. I've removed it completely and the build is passing now. Thanks for catching that.

**Pull request merged**